Quantum-Control-Applications/Superconducting/Multi Flux Tunable transmon/Use Case 1 - Two qubit gate optimization with cryoscope/readme.md

```diff
@@ -0,0 +1,188 @@
+
+## 1. Experimental set-up and required calibrations
+
+
+<img align="right" src="setup.PNG" alt="drawing" width="400"/>
+
+### 1.1 Experimental set-up
+The experimental setup consists of two flux tunable transmons (with a SQUID loop as nonlinear inductor and a flux line next to it) coupled to readout resonators located on a common transmission line.
+The general scheme is represented on the figure on the right.
+
+The qubits are controlled with IQ signals generated by the OPX and upconverted with an external local oscillator using calibrated IQ mixers.
+
+The readout resonators are pumped with an intermediate frequency generated by the OPX and upconverted with another external local oscillator using an IQ mixer.
+The reflected signal is measured by the OPX after downconversion with an IQ mixer and the local oscillator signal.
+
+The flux lines are controlled by two additional analog outputs of the OPX.
+
+In principle, the program detailed below can be easily modified if a different set-up is to be used.
+
+### 1.2 Calibration steps prior to the cryoscope and two-qubit chevron experiments
+
+Before running Cryoscope, several calibration steps are required.
+
+* The first calibration to run is the 2D readout resonator spectroscopy (flux amplitude and frequency). This spectroscopy will show the resonator frequency as a function of flux biasing, the qubit flux insensitive point (qubit frequency independent of flux bias), the qubit zero-frequency point (where the flux bias is ...), and the avoided crossing when the qubit and resonator have the same frequency. For Cryoscope, we want to set the flux line to the qubit flux insensitive point and bias it around $\phi_0/4$ during the sequence.   
+* The second important quantity to calibrate is the $\pi/2$ pulse. This can be done using a simple Rabi sequence where both the amplitude and frequency are scanned while the flux bias is set to the qubit flux insensitive point found previously.
+* (optional) If the qubit pulses are relatively short (< 100ns), then the use of optimal DRAG waveforms can enhance the fidelity of the gates [[3]](#3). 
+* The last calibration to perform is the qubit detuning versus flux bias. This will be used to validate the qubit flux insensitive point and the avoided level crossing, as well as the cryoscope measurement. The sequence can be found in the [Cryoscope use-case](#2).
+
+## 2. The configuration file
+
+### 3.1 The elements
+The configuration consists of 3 elements:
+* `qubit0` and `qubit1` send control pulses to each transmon
+* `resonator0` and `resonator1` send pump pulses and measures the reflected signal for each qubit
+* `flux_line0` and `flux_line1` send control pulses and DC voltage to the flux line of each qubit
+
+
+### 3.2 The operations, pulse and waveforms
+The qubits elements `qubit0` and `qubit1` have several operations defined that correspond to X/2, Y/2 and X gates.

```

Comment:
maybe also list `X/2, Y/2, Y and X gates`

Reply:
Ok, but we don't actually use the Y gate here